setup/productionize/caching/externalize-redis.md

```diff
@@ -64,3 +64,18 @@ services.redis.baseUrl: $REDIS_ENDPOINT
 
 > `$SERVICE` is the service name (e.g. `clouddriver`) that is being configured
 > to use another endpoint.
+
+## Using a hosted Redis
+
+Gate requires keyspace notifications to be enabled in Redis, and tries to configure
+this when it starts up. Some hosted Redis services disable the `CONFIG` command, blocking
+Gate from modifying the configuration. In this case:
+1. Manually set the configuration parameter `notify-keyspace-events` to `gxE` on your Redis
+instance by following the documentation provided by your hosted Redis provider.
```

Comment:
Is this normally done via a config file for redis? redis-cli? How's it done for Cloud Memorystore?

Reply:
Yes, I believe that with a user-installed Redis you'd normally set it as a config parameter.  For Cloud Memorystore, the documentation is here https://cloud.google.com/memorystore/docs/redis/configuring-redis.  (Configuration parameters can also be edited in the UI for Cloud Memorystore, which doesn't seem to be in the link.)  I wanted to avoid getting into details of specific Redis offerings here, but let me know if you think additional details would be helpful.